readme.md

```diff
@@ -302,11 +302,23 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [upf-bpf](https://github.com/navarrothiago/upf-bpf) - An in-kernel solution based on XDP for 5G UPF.
 - [redbpf](https://github.com/foniod/redbpf) - Tooling and framework to write eBPF code in Rust efficiently.
 - [pixie](https://github.com/pixie-io/pixie) - Observability for Kubernetes using eBPF. Features include protocol tracing, application profiling, and support for distributed bpftrace deployments.
-- [Falco](https://falco.org/) - Falco, the cloud-native runtime security project, is the de facto Kubernetes threat detection engine
-- [Sysmon for Linux](https://github.com/Sysinternals/SysmonForLinux) - Sysmon For Linux is a security monitoring tool. It depends on [SysinternalsEBPF](https://github.com/Sysinternals/SysinternalsEBPF).
+- [Falco](https://falco.org/) - Falco is a cloud-native runtime security project used as a Kubernetes threat detection engine.
```

Comment:
Since we have Falco, would you mind removing Sysdig from the list (above), please? Unless I'm mistaken they're doing eBPF through Falco, they contributed the code to the CNCF some time ago.

Reply:
Ack, will remove the link to the more generic Sysdig tracing tool above which now use Falco as their eBPF libs. In the latest release of Sysdig they also state that

> This is the first Sysdig release to make full use of the Falco Libs since its donation to the CNCF in 2021.